src/data/finset/basic.lean

```diff
@@ -897,6 +897,14 @@ subset_insert_iff.1 $ subset.refl _
 theorem insert_erase_subset (a : α) (s : finset α) : s ⊆ insert a (erase s a) :=
 subset_insert_iff.2 $ subset.refl _
 
+lemma erase_inj_of_mem {x y : α} (s : finset α) (hx : x ∈ s)
+  (h : s.erase x = s.erase y) :
+  x = y :=
+begin
```

Comment:
Sorry for not catching this last time - usually `_inj` lemmas have an iff in their goal statement. I know the reverse direction is trivial, but I think its worth having to make this work in `rw`:
```suggestion
lemma erase_inj_of_mem {x y : α} (s : finset α) (hx : x ∈ s) :
  s.erase x = s.erase y ↔ x = y :=
begin
  refine ⟨λ h, _, congr_arg _⟩,
```

Reply:
Done.